### ISSUE TITLE
Bad controllability for box condition.

### ISSUE THREAD

**Issue by swc-17** [cure-lab/MagicDrive#102]
Hi, thanks for your great work. 
I build a model sharing the same camera/box encoder with Magicdrive, without map condition and BEVControlNet, and set all params in UNet trainable. After trained for 100 epochs, I found the generated images do not match with box condition well, for some boxes, the images do not have foreground objects at the corresponding location. Any suggestions on this? And I wonder how many epochs is the released model trained for? 350e as in the config? Thanks!

**Comment by swc-17** [comment]
![clipboard-image-1730862149](https://github.com/user-attachments/assets/0949e0f4-e92b-4358-bff8-15b3ec6b460a)

**Comment by flymin** [comment]
> without map condition and BEVControlNet, and set all params in UNet trainable.

We never tried to train like this. You may adjust the parameters, like learning rate or batch size, to optimize the training process.

Typically, 100 epochs should be fine in most cases. The release model is trained with 350 epochs to optimize the results for quantitative evaluation.

**Comment by swc-17** [comment]
Thanks for your reply!
